packages/slice-machine/src/theme.ts

```diff
@@ -68,7 +68,7 @@ const AppTheme = (): Theme =>
       warning: "#E67E22",
       warning02: "#ED811C",
       sidebar: "#F1F1F4",
-      link: "#5163BA",
+      link: "#6E56CF",
```

Comment:
❓ This doesn't cause side-effects elsewhere?

Reply:
It's actually used nowhere but in my component :/